api/test/trace/propagation/http_text_format_test.cc

```diff
@@ -0,0 +1,206 @@
+#include "opentelemetry/context/context.h"
+#include "opentelemetry/nostd/shared_ptr.h"
+#include "opentelemetry/nostd/span.h"
+#include "opentelemetry/nostd/string_view.h"
+#include "opentelemetry/trace/default_span.h"
+#include "opentelemetry/trace/span.h"
+#include "opentelemetry/trace/span_context.h"
+#include "opentelemetry/trace/trace_id.h"
+#include "opentelemetry/trace/tracer.h"
+
+#include <map>
+#include <memory>
+#include <string>
+
+#include <gtest/gtest.h>
+
+#include "opentelemetry/trace/default_span.h"
+#include "opentelemetry/trace/propagation/http_text_format.h"
+#include "opentelemetry/trace/propagation/http_trace_context.h"
+
+using namespace opentelemetry;
+
+static nostd::string_view Getter(const std::map<std::string, std::string> &carrier,
+                                 nostd::string_view trace_type = "traceparent")
+{
+  auto it = carrier.find(std::string(trace_type));
+  if (it != carrier.end())
+  {
+    return nostd::string_view(it->second);
+  }
+  return "";
+}
+
+static void Setter(std::map<std::string, std::string> &carrier,
+                   nostd::string_view trace_type        = "traceparent",
+                   nostd::string_view trace_description = "")
+{
+  carrier[std::string(trace_type)] = std::string(trace_description);
+}
+
+static trace::propagation::HttpTraceContext<std::map<std::string, std::string>> format =
+    trace::propagation::HttpTraceContext<std::map<std::string, std::string>>();
+
+static const nostd::string_view trace_id = "12345678901234567890123456789012";
+static const nostd::string_view span_id  = "1234567890123456";
```

Comment:
The style guide calls for naming constants like kConstantName or CONSTANT_NAME.

```suggestion
static const nostd::string_view kTraceId = "12345678901234567890123456789012";
static const nostd::string_view kSpanId  = "1234567890123456";
```

Reply:
After reviewing it, I actually found these two are not being used anywhere, so I removed them instead